qutebrowser/config/configdata.yml

```diff
@@ -1191,9 +1191,14 @@ prompt.radius:
 ## scrolling
 
 scrolling.bar:
-  type: Bool
-  default: false
-  desc: Show a scrollbar.
+  type:
+    name: String
+    valid_values:
+      - always: Always show the scrollbar.
+      - never: Never show the scrollbar.
+      - when_searching: Show the scrollbar when searching for text in the webpage.
```

Comment:
This should be `when-searching` to be consistent with other option values, changed in 6d0d40878589a2c9f5bb90848cfd00ef9dd4035f